src/actions/run.ts

```diff
@@ -52,5 +53,8 @@ export async function generateCliCommand(
     for (const path of paths) {
         command += ` --uploadDir=${path}`
     }
+    for (const tag of tags || []) {
+        command += ` --tags=${tag}`
```

Comment:
Since tags are passed one by one and each option represents a single tag with key and value, I would prefer using `--tag=${tag}` instead of `--tags=${tag}`